Code/Engine/RendererFoundation/CommandEncoder/CommandEncoder.h

```diff
@@ -5,11 +5,16 @@
 #include <RendererFoundation/CommandEncoder/CommandEncoderPlatformInterface.h>
 #include <RendererFoundation/CommandEncoder/CommandEncoderState.h>
 
+struct ezGALRenderingSetup;
+
 class EZ_RENDERERFOUNDATION_DLL ezGALCommandEncoder
 {
   EZ_DISALLOW_COPY_AND_ASSIGN(ezGALCommandEncoder);
 
 public:
+  ezGALCommandEncoder(ezGALDevice& in_device, ezGALCommandEncoderCommonPlatformInterface& in_commonImpl);
```

Comment:
We don't use in_ prefix anywhere else

Reply:
I was forced to pick one out of in / out / ref by clangTidy, gosh so many options, hmm ref?